### PR TITLE
boswars: new recipe

### PR DESCRIPTION
**Pull request by Crestwave**

### CODE REVIEW

File: games-strategy/boswars/boswars-2.7.recipe [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	boswars$secondaryArchSuffix = $portVersion
+	app:BosWars = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
```
Comment (return) [inline games-strategy/boswars/boswars-2.7.recipe]: This should be in alphabetical order `lib:libGL$secondaryArchSuffix` placed first over `lib:libintl$secondaryArchSuffix`.

File: games-strategy/boswars/boswars-2.7.recipe [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python2
+	"
+
+defineDebugInfoPackage boswars$secondaryArchSuffix \
+	"$appsDir/BosWars"
+
+PATCH()
+{
+	set -- maps/antarticum.map/terrain.png graphics/ui/ui_*.png \
```
Comment (return) [inline games-strategy/boswars/boswars-2.7.recipe]: Could you describe in a comment on what this PATCH function is doing?

File: games-strategy/boswars/boswars-2.7.recipe (hunk 1 above)
Comment (Crestwave) [reply games-strategy/boswars/boswars-2.7.recipe]: Whoops, I forgot to clean that part up. It doesn't even link against `libintl` IIRC

File: games-strategy/boswars/boswars-2.7.recipe (hunk 2 above)
Comment (Crestwave) [reply games-strategy/boswars/boswars-2.7.recipe]: Is the current comment enough or do you want me to describe what each section does?

Comment (return) [reply games-strategy/boswars/boswars-2.7.recipe]: The current comment (which you have added in your new commit) is enough to describe this PATCH function.

File: games-strategy/boswars/boswars-2.7.recipe [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+PROVIDES="
+	boswars$secondaryArchSuffix = $portVersion
+	app:BosWars = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:liblua$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
```
Comment (Begasus) [inline games-strategy/boswars/boswars-2.7.recipe]: libpng16 (see https://github.com/haikuports/haikuports/issues/3525) (same for devel*)

File: games-strategy/boswars/boswars-2.7.recipe (hunk 1 above)
Comment (Crestwave) [reply games-strategy/boswars/boswars-2.7.recipe]: Fixed. The funny thing is that I was actually considering to use `libpng16` but then I looked into its recipe and found that it also provided `libpng`. A comment should probably be added there.

File: games-strategy/boswars/boswars-2.7.recipe (hunk 3 above)
Comment (Crestwave) [reply games-strategy/boswars/boswars-2.7.recipe]: Huh, my comment earlier didn't go through. Basically, I was actually going to use `libpng16` but then I checked the recipe and found that there was actually a `PROVIDES` for `libpng`. A command should probably be added there.

File: games-strategy/boswars/boswars-2.7.recipe [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libGLU$secondaryArchSuffix
+	devel:liblua5.1$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libSDL$secondaryArchSuffix
```
Comment (Begasus) [inline games-strategy/boswars/boswars-2.7.recipe]: Missed this one (my bad), should be libSDL_1.2 :)

Comment (Crestwave) [reply games-strategy/boswars/boswars-2.7.recipe]: I'm not familiar with adding changes after a pull request is merged. Does GitHub have a feature to merge the later changes? Most of my recent PRs made this mistake so I could make a PR with fixes for all of them if that's preferable.

Comment (pulkomandy) [reply games-strategy/boswars/boswars-2.7.recipe]: Just make a new PR with the fixes. And yes, all of them in one single go will be fine.

Comment (Begasus) [reply games-strategy/boswars/boswars-2.7.recipe]: If you do, be sure to bump revision for them

**Pull request merged**